presto-cli/src/main/java/com/facebook/presto/cli/StatusPrinter.java

```diff
@@ -443,4 +458,33 @@ private static double percentage(double count, double total)
         }
         return min(100, (count * 100.0) / total);
     }
+
+    private class PreviousStageStatus
```

Comment:
This is not a great name. There's nothing "previous" about the contents of this class. It's just "stage stats". Whether it it's used to track the stats of the previous update is contextual.